### DISCUSSION TITLE
In frontend, all my devices don't report Description Last Seen

### DISCUSSION THREAD

**Discussion by acaonweb** [Koenkk/zigbee2mqtt#16155] [General]
I have about 30 devices from many different vendors, but in everyone of them I have that
<img width="1314" alt="Screenshot 2023-01-14 alle 09 44 17" src="https://user-images.githubusercontent.com/22882393/212464058-c9c4ecdf-1fe7-477a-9fa1-6f25c240098a.png">

It's a normal behavior?

**Comment by OrbitingOcelot** [comment]
Go to: settings -> advanced -> last seen, and make sure it is set correctly (i.e. not "disabled"). Does that help?

**Reply by acaonweb** [reply]
Bingo!!!
Resolved. 
Thanks
